dev/bots/suite_runners/run_customer_testing_tests.dart

```diff
@@ -30,7 +30,9 @@ Future<void> customerTestingRunner() async {
     ],
     workingDirectory: flutterRoot,
   );
-  if (Platform.environment case {'REVISION': final String revision}) {
+  final Map<String, String> env = Platform.environment;
```

Comment:
this line could be removed

Reply:
I think you're looking at a single commit, which is reverting a few things.

**run_customer_testing_tests.dart** is no longer being changed in this PR :)

Reply:
This is a revert but line 33 seems unnecessary as there doesn't seem to be any need to put Platform.environment in a variable and can be accessed directly since it's not accessed multiple times. This is how it is done in other code blocks too.